docs/how-to-use-kit.md

```diff
@@ -0,0 +1,105 @@
+* Metrics that we collect from EKS clusters are limited to API server metrics and CW metrics, we don’t get the node level metrics from EKS control plane nodes. In guest clusters, we are able to collect API server, etcd, scheduler, KCM and node level metrics.
+* Guest clusters do not support testing Managed node groups or Fargate or IRSA at this point.
+
+
+### How to run tests
+
+Add tasks and pipelines from the KIT repo, (we have a [task](https://github.com/awslabs/kubernetes-iteration-toolkit/issues/207) pending to automate this step)
+
+```
+# cd github.com/awslabs/kubernetes-iteration-toolkit/tests
+# kubectl apply -n tekton-pipelines -f pipelines -R
+# kubectl apply -n tekton-pipelines -f tasks -R
+```
+
+* Access the Tekton dashboard on [http://localhost:9097](http://localhost:9097/), make sure you have run port-forward command mentioned above.
+* Under the tasks tab look for the pre-loaded tasks that can be run.
+* Under pipelines tab are some sample pipelines flow which help create logic regarding run multiple tasks for an end-to-end test flow. Example pipeline combines the following tasks-
+    * Create an EKS cluster + create a managed node group with `n` nodes + deploy `n` pods in the cluster + measure  pod startup latency as part of the test
+* These pipelines can be triggered from dashboard using pipeline runs or by creating a CRD for pipeline run.
+* Based on these samples, a users can create their own pipelines for specific tasks based on their test use case and run these pipelines using pipeline runs.
+
+
+How to collect metrics
+
+* Access the Grafana dashboard on [http://localhost:8080](http://localhost:8080/), make sure you have run port-forward command mentioned above.
+* All the core Kubernetes components like API Server, KCM, scheduler, etcd, authenticator are being scraped and their metrics are being plotted in their own Grafana dashboards
+* For node level metrics in Grafana refer to NodeExporter/Nodes dashboard
+
+Adding additional dashboards from Github repo-
+
+```bash
+# wget grafana dashboard from the repo
+wget https://raw.githubusercontent.com/awslabs/kubernetes-iteration-toolkit/main/substrate/monitoring/master-dashboard.json
+
+# Create configmap in management cluster to load this dashboard
+kubectl create configmap master-dashboard -n monitoring --from-file=master-dashboard.json
+kubectl label configmap master-dashboard -n monitoring  grafana_dashboard=1
+```
+>Note: we are [tracking the task](https://github.com/awslabs/kubernetes-iteration-toolkit/issues/207) on automating this step as well where the Grafana dashboards are pre-loaded in Grafana
+
+Accessing logs for kube components-
+[WIP](https://github.com/awslabs/kubernetes-iteration-toolkit/issues/106)
+
+### Use-cases supported with KIT
+
+* As a Kubernetes developer/user, create x number of objects(pods, secrets, config maps) in a cluster. As part of the validation, I want to
```

Comment:
Could this section be a little more concise? For example I don't think we need to prefix each bullet with "As a Kubernetes developer/user."